drivers/wifi/nrf700x/Kconfig

```diff
@@ -69,27 +76,23 @@ config NRF700X_AP_MODE
 
 config NRF700X_P2P_MODE
 	bool "Enable P2P support in driver"
-
-endif # NRF700X_SYSTEM_MODE
-
-config NRF700X_DATA_TX
-	bool "Enable TX data path in the driver"
-	default y if NRF700X_SYSTEM_MODE
+endif # NRF700X_SYSTEM_MODE || NRF700X_SYSTEM_WITH_RAW_MODES
 
 config NRF700X_RAW_DATA_TX
 	bool "Enable RAW TX data path in the driver"
 	select EXPERIMENTAL
-	depends on NRF700X_SYSTEM_MODE
 
 config NRF700X_RAW_DATA_RX
 	bool "Enable RAW RX sniffer operation in the driver"
 	select EXPERIMENTAL
-	depends on NRF700X_SYSTEM_MODE
 
 config NRF700X_PROMISC_DATA_RX
 	bool "Enable promiscuous RX sniffer operation in the driver"
 	select EXPERIMENTAL
-	depends on NRF700X_SYSTEM_MODE
+
+config NRF700X_DATA_TX
```

Comment:
Any reason this is moved outside the `if NRF700X_SYSTEM_MODE || NRF700X_SYSTEM_WITH_RAW_MODES`